spec/other/travis_spec.rb

```diff
@@ -1,11 +1,19 @@
 describe 'travis.yml' do
-  it "matches versions and excludes" do
-    travis = YAML.safe_load(File.open(ManageIQ::UI::Classic::Engine.root.join('.travis.yml')))
+  let(:travis) { YAML.safe_load(File.open(ManageIQ::UI::Classic::Engine.root.join('.travis.yml'))) }
+  let(:versions) { travis['rvm'] }
 
-    versions = travis['rvm']
-    excludes = travis.dig('matrix', 'exclude').map { |ex| ex['rvm'] }.sort.uniq
+  it "matches versions and excludes for multiple ruby versions" do
+    if versions.length > 1
+      excludes = travis.dig('matrix', 'exclude').map { |ex| ex['rvm'] }.sort.uniq
+      expect(excludes.length).to eq(versions.length - 1)
+      expect(versions).to include(excludes.first)
```

Comment:
> `expect(versions.first).to eq(excludes.first)`

The idea behind ^ was to make sure we're excluding the old version, not the new.

I guess we would want to use the newest one with more 2 versions.

So, maybe something like..

`expect(versions[1..-1]).to eq(excludes)`

?

Reply:
But `versions` contains something like `['2.4.6', '2.5.3']`. So I guess `expect(versions[0..-2]).to eq(excludes)`, right?

Reply:
Sorry, you're right, versions go from oldest, not newest :+1:

Reply:
Fixed.